mentat/app.py

```diff
@@ -35,10 +36,35 @@ def run_cli():
         default=[],
         help="List of file paths, directory paths, or glob patterns to exclude",
     )
+    parser.add_argument(
+        "--no-backup",
+        action="store_false",
+        help="No backup mode for the mentat",
+    )
+    parser.add_argument(
+        "--backup-dir",
+        type=str,
+        default=".mentat_backups",
+        help="Directory to store backups, if not provided, backups will be stored in the .mentat_backups directory",
+    )
```

Comment:
let's remove these options to keep things simple and nimble for now. We'll just always backup (as I don't see a downside) and always use `.mentat_backups` to store the backup